FRENCH/src/ch07-00-managing-growing-projects-with-packages-crates-and-modules.md

```diff
@@ -0,0 +1,120 @@
+<!--
+# Managing Growing Projects with Packages, Crates, and Modules
+-->
+
+# Gérer des projets grandissants avec les paquets, crates et modules
+
+<!--
+As you write large programs, organizing your code will be important because
+keeping track of your entire program in your head will become impossible. By
+grouping related functionality and separating code with distinct features,
+you’ll clarify where to find code that implements a particular feature and
+where to go to change how a feature works.
+-->
+
+Lorsque vous commencez à écrire des gros programmes, organiser votre code va
+devenir important car vous ne pourrez plus vous souvenir de l'intégralité de
+votre programme de tête. En regroupant des fonctionnalités qui ont des points
+communs et en séparant des autres fonctionnalités, vous clarifiez l'endroit où
+trouver le code qui implémente une fonctionnalité spécifique et où aller pour
+changer une fonctionnalité.
+
+<!--
+The programs we’ve written so far have been in one module in one file. As a
+project grows, you can organize code by splitting it into multiple modules and
+then multiple files. A package can contain multiple binary crates and
+optionally one library crate. As a package grows, you can extract parts into
+separate crates that become external dependencies. This chapter covers all
+these techniques. For very large projects of a set of interrelated packages
+that evolve together, Cargo provides workspaces, which we’ll cover in the
+[“Cargo Workspaces”][workspaces]<!-- ignore -- > section in Chapter 14.
+-->
+
+Le programme que nous avons écrit précédemment était dans un module et dans un
+seul fichier. Quand le projet grandit, vous pouvez organiser votre code en le
+découpant en plusieurs modules et ensuite en plusieurs fichiers. Un paquet peut
+contenir plusieurs crates binaires et accessoirement une crate de bibliothèque.
```

Comment:
`crate binaire` and `crate de bibliothèque` don't reflect what they are.
- `binary crate` refers to runnable code
- `library crate` refers to code meant to be used in runnable code

As this is a term used in the previous chapters, this should probably be fixed in a seperate PR